dbt/models/iasworld/schema/iasworld.aprval.yml

```diff
@@ -17,12 +17,35 @@ sources:
             description: Applied market value
           - name: aprbldg
             description: Appraised building value
+            tests:
+              - dbt_utils.accepted_range:
+                  name: iasworld_aprval_aprbldg_between_0_and_1B
+                  min_value: 0
+                  max_value: 1000000000
+                  config:
+                    where: |
+                      taxyr >= '2021'
+                      AND cur = 'Y'
+                      AND deactivat IS NULL
           - name: aprdate
             description: '`APRTOT` calc date'
           - name: aprland
             description: Appraised land value
+            tests:
+              - dbt_utils.accepted_range:
+                  name: iasworld_aprval_aprland_between_0_and_1B
+                  min_value: 0
+                  max_value: 1000000000
+                  config: &unique-conditions
+                    where: cur = 'Y' AND deactivat IS NULL
```

Comment:
Since many tests require the same conditionals, I used YAML anchors a lot to repeat the same conditions across tests in the same schema file.